src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -24,7 +24,6 @@ import * as User from '../../../../libs/actions/User';
 import * as ReportUtils from '../../../../libs/ReportUtils';
 import * as ReportActionsUtils from '../../../../libs/ReportActionsUtils';
 import canFocusInputOnScreenFocus from '../../../../libs/canFocusInputOnScreenFocus';
-import debouncedSaveReportComment from '../../../../libs/ComposerUtils/debouncedSaveReportComment';
```

Comment:
@DylanDylann 
We also need to remove the stale function code as well, right? I assume its used nowhere else.

Reply:
Sure. Will remove this stale function

Reply:
@abdulrahuman5196 just removed it. Please help check